includes/class-sensei-grading-user-quiz.php

```diff
@@ -182,27 +189,33 @@ public function display() {
 				$graded_class        = '';
 				$user_question_grade = Sensei()->quiz->get_user_question_grade( $lesson_id, $question_id, $user_id );
 				$graded_class        = 'ungraded';
-				if ( 0 == $question_grade_total && 0 == intval( $user_question_grade ) ) {
-					// Question skips grading
-					$grade_type   = 'zero-graded';
-					$graded_class = '';
-					++$correct_answers;
-					++$graded_count;
+
+				// Question with no grade value associated with it.
+				if ( 0 === $question_grade_total ) {
+					$grade_type          = 'zero-graded';
+					$graded_class        = '';
 					$user_question_grade = 0;
-				} elseif ( intval( $user_question_grade ) > 0 ) {
-					$graded_class = 'user_right';
-					++$correct_answers;
-					$user_quiz_grade_total += $user_question_grade;
-					++$graded_count;
+					$graded_count++;
 				} else {
-					if ( ! is_string( $user_question_grade ) && intval( $user_question_grade ) == 0 ) {
-						$graded_class = 'user_wrong';
-						++$graded_count;
+					$user_right = intval( $user_question_grade > 0 );
```

Comment:
Did you mean to change it to convert from `bool` to `int` here or is the parens misplaced? Seems like we more want to just cast `$user_question_grade` to `int` in case it is something else.

Reply:
Nope, that's a typo. Fixed in https://github.com/Automattic/sensei/pull/2816/commits/bed01773e842dc15f5d8e66ec52978a365a6ae67.